llvm/lib/Target/RISCV/RISCVSubtarget.cpp

```diff
@@ -62,10 +62,15 @@ static cl::opt<unsigned> RISCVMinimumJumpTableEntries(
     "riscv-min-jump-table-entries", cl::Hidden,
     cl::desc("Set minimum number of entries to use a jump table on RISCV"));
 
-static cl::opt<bool>
-    UseMIPSLoadStorePairsOpt("mips-riscv-load-store-pairs",
-                             cl::desc("RISCV: Optimize for load-store bonding"),
-                             cl::init(false), cl::Hidden);
+static cl::opt<bool> UseMIPSLoadStorePairsOpt(
+    "mips-riscv-load-store-pairs",
+    cl::desc("RISCV: Enable the load/store pair optimization pass"),
```

Comment:
I don't think "RISCV" is needed in this descritpion since its already in the option name. But if was, it should always be spelled "RISC-V" per RVI branding guidelines.